noxfile.py

```diff
@@ -734,6 +734,15 @@ def _pytest(session, coverage, cmd_args):
             check=True,
         )
 
+    if hasattr(platform, "dist"):
+        if platform.dist()[0] == "debian" and platform.dist()[1].startswith("10"):
```

Comment:
Now that Debian 11 is oldstable, might this also apply to that version of Debian

Reply:
This is just working around an issue with the debian 10 image, since its more difficult to updte the old golden images since we are now using a new process. This only affects debian 10 right now.